loc/ui/stats.js

```diff
@@ -157,63 +158,72 @@ export default {
       en: 'Attack Spd.',
       ru: 'Скорость Атк.',
       fr: 'Vit. Attq',
-      pt: 'Rapidez Atq.'
+      pt: 'Rapidez Atq.',
+      tr: 'Saldırı Hızı'
     },
 
     {
       en: 'Item Find',
       ru: 'Выпадение',
       fr: 'Trouvaille',
-      pt: 'Encontrar Item'
+      pt: 'Encontrar Item',
+      tr: 'Çğe Bulma'
     },
 
     {
       en: 'Bag Slots',
       ru: 'Слоты Сумки',
       pt: 'Espaço',
-      fr: 'Espace Sac'
+      fr: 'Espace Sac',
+      tr: 'Çanta Yuvaları'
```

Comment:
The translations must not be any more than 2 characters longer than the original English string. Please shorten this.